src/components/HiddenTaskList/HiddenTaskList.jsx

```diff
@@ -70,14 +69,15 @@ class HiddenTaskList extends React.Component {
 
 HiddenTaskList.propTypes = {
   tasks: React.PropTypes.array.isRequired,
+  activeFilter: React.PropTypes.object.isRequired,
```

Comment:
On the react side it is consistent. There is one spot on the redux side where it wasn't consistent, but I just committed a change to fix that.